push-files/push.js

```diff
@@ -28,30 +28,37 @@ async function buildMetadata(input) {
   return data;
 }
 
-// getHashOfFiles is a badly implemented way of hashing the file contents. The tar
-// isn't reproducible across different systems, so this is pretty pointless.
 async function getHashOfFiles(filesDir) {
-  let tard = '';
-  const options = {
-    silent: true,
-    cwd: filesDir,
-  };
-  options.listeners = {
-    stdout: (data) => {
-      tard += data.toString();
-    },
-  };
+  return new Promise((resolve, reject) => {
+    const walker = walk.walk(`${filesDir}`);
```

Comment:
Any reason that's not just `walk.walk(filesDir);`?

Reply:
No, not really. Thanks for the hint.